driver-core/src/main/com/mongodb/operation/MixedBulkWriteOperation.java

```diff
@@ -221,19 +243,25 @@ private boolean serverSupportsWriteCommands(final ConnectionDescription connecti
     }
 
     private Iterable<Run> getRunGenerator(final ConnectionDescription connectionDescription) {
+        Boolean bypassValidation = null;
```

Comment:
Good catch - it was originally in other operations then removed, I missed this one, will remove.